src/main/java/com/google/devtools/build/lib/cmdline/IgnoredSubdirectories.java

```diff
@@ -29,42 +41,98 @@
  * <p>This is currently just a prefix, but will eventually support glob-style wildcards.
```

Comment:
Can be updated now that globs are supported.

Reply:
Done (not exported yet, though, stay tuned)